src/app/header/context-help-toggle/context-help-toggle.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ContextHelpService } from '../../shared/context-help.service';
+import { TranslateService } from '@ngx-translate/core';
+import { Observable, Subscription, BehaviorSubject } from 'rxjs';
+import { combineLatest } from 'rxjs';
+
+@Component({
+  selector: 'ds-context-help-toggle',
+  templateUrl: './context-help-toggle.component.html',
+  styleUrls: ['./context-help-toggle.component.scss']
+})
+export class ContextHelpToggleComponent implements OnInit, OnDestroy {
+  buttonDisabled$: Observable<boolean>;
+
+  constructor(
+    private contextHelpService: ContextHelpService,
+    private translateService: TranslateService
```

Comment:
it seems to be not used, could you remove it ?

Reply:
Thanks for the feedback! I removed the `translateService` parameter from the constructor.